ush/get_extrn_mdl_file_dir_info.sh

```diff
@@ -399,7 +399,8 @@ fi
         if [ "${MACHINE}" = "JET" ] || [ "${MACHINE}" = "ORION" ]; then
           fns_on_disk=( "${yy}${ddd}${hh}0${fcst_mn}0${fcst_hh}" )
         elif [ "${MACHINE}" = "HERA" ] ; then
-          fns_on_disk=( "gfs.t${hh}z.pgrb2.0p25.f0${fcst_hh}" )
+          #fns_on_disk=( "gfs.t${hh}z.pgrb2.0p25.f0${fcst_hh}" )
+          fns_on_disk=( "${yy}${ddd}${hh}0${fcst_mn}0${fcst_hh}" )
```

Comment:
These changes will only work for the staged GFS data that were copied from JET. However, the real-time GFS data on HERA are not using this format.

Reply:
Right now, we only do retro on hera. We need make sure retro can be run without too much efforts. We will change it back if we want to run real-time on Hera.

Reply:
Okay, just want to make sure we have this in mind.